### ISSUE TITLE
When using NVDA, tabbing to the "Browse" faux-link announces "blank"

### ISSUE THREAD

**Issue by Vuice** [pqina/filepond#247]
I am currently testing FilePond for accessibility by using NVDA on Chrome using strictly a keyboard.  When visiting your site (https://pqina.nl/filepond/) and removing the default file that is loaded in the uploader, I use the keyboard to tab from the top of the page to the uploader.  Below is the output from the NVDA speech viewer:

> Download
> visited  link
> Examples
> visited  link
> Features
> visited  link
> Roadmap
> visited  link
> Plugins
> visited  link
> Docs
> visited  link
> main landmark 
> Drag & Drop your files or Browse
> button
> **blank**
> clickable
> main landmark 
> FilePond on GitHub
> 5,094
> visited  link

The "blank" is announced when the "Browse" text contained in the span with class "filepond--label-action" is the focus (with the highlight around the text).  I know coding for accessibility is not easy, but I was wondering why the "Browse" faux-link cannot just be a button (like the "remove" or "upload" buttons in the file items) so that it can be announced correctly by screen readers?

In addition to this, when tabbing through with a keyboard, "Drag & Drop your files or Browse" text and "blank" still will be announced when it is hidden from view by the file item that is taking its place.  This also needs to not be reachable by screen readers when it is supposed to be hidden.  The issue, in its current state, is that the tabindex is still equal to "0" instead of "-1" when hidden from view.

P. S. I love this uploader!

**Comment by rikschennink** [comment]
Hi! Thanks so much for reporting this, I've tabbed through FilePond using VoiceOver and NVDA but did not catch this, I try to make FilePond as accessible as possible but I might have missed a couple things (I obviously did :D) I'll see if I can fix these issues over the next couple of weeks.

**Comment by rikschennink** [comment]
Just ran some tests.

It seems to pronounce `"Drag & Drop your files or Browse button"` correctly, sometimes it says `"blank clickable"` and sometimes it does not. `aria-hidden` or other `role=button` don't seem to work to persuade it to do otherwise. Also setting a different label with `aria-label` does not work.

It's a `span` instead of a `button` because the element is contained inside a `label` that already serves as a button to focus/activate the related `input` file element. In this case I only wanted to highlight the _browse_ part when tabbing through the page as you can't drag and drop files with your keyboard.

Users can still drop/browse files when the label is moved to the top/visuall hidden. It's only hidden from an visual point of view. So it's still active on purpose.

Happy to make changes if you can find a workable solution, will close otherwise as I think even with the "clickable blank" part in there the component is still very usable.
